Mathlib/AlgebraicGeometry/Morphisms/Smooth.lean

```diff
@@ -165,4 +166,17 @@ instance : MorphismProperty.IsMultiplicative (@IsSmoothOfRelativeDimension 0) wh
   id_mem _ := inferInstance
   comp_mem _ _ _ _ := inferInstance
 
+/-- Smooth morphisms are of finite presentation. -/
+instance (priority := 100) [hf : IsSmooth f] : LocallyOfFinitePresentation f := by
+  rw [HasRingHomProperty.eq_affineLocally @LocallyOfFinitePresentation]
+  rw [HasRingHomProperty.eq_affineLocally @IsSmooth] at hf
+  refine affineLocally_le (fun hf ↦ ?_) f hf
+  apply RingHom.locally_of_locally (Q := RingHom.FinitePresentation) at hf
+  · rwa [RingHom.locally_iff_of_localizationSpanTarget finitePresentation_respectsIso
+      finitePresentation_ofLocalizationSpanTarget] at hf
+  · introv hf
+    algebraize [f]
+    haveI : Algebra.IsStandardSmooth R S := hf
```

Comment:
@callesonne do you understand why `algebraize` does not generate this instance?

Reply:
LGTM, modulo this issue. So maybe
```suggestion
    algebraize [f]
    -- TODO: why is `algebraize` not generating the following instance?
    haveI : Algebra.IsStandardSmooth R S := hf
```